### PR TITLE
Import script for Melton (2024-05-02) (closes #6268)

### LINKED ISSUE #6268: Import MEL-MEL for 2024-05-02
EMS: Xpress DC
Files:
- `MEL/2024-05-02/2024-02-23T09:32:15.458646/Democracy_Club__02May2024.tsv`

### PR DESCRIPTION
**Pull request by polling-bot-4000**
PR triggered by upload at https://github.com/DemocracyClub/UK-Polling-Stations/issues/6268

### CODE REVIEW

**Comment by awdem** [comment]
https://app.asana.com/0/1205951895246773/1206713471172149/f

File: polling_stations/apps/data_importers/management/commands/import_melton.py [hunk 1]
```diff
@@ -4,18 +4,33 @@
...
+        # corrections from council:
+        # The Old School, 4 Main Street, Muston, Nottingham
+        if record.polling_place_id == "2694":
+            record = record._replace(
+                polling_place_uprn="100030550160",
+                polling_place_easting="482883",
+                polling_place_northing="337945",
+                polling_place_postcode="NG13 0FB",
+            )
+            print(record)
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_melton.py:25]: Could you delete this line.

**Review by GeoWill — COMMENTED** [review]
All good. Just a stray `print`

File: polling_stations/apps/data_importers/management/commands/import_melton.py (hunk 1 above)
Comment (awdem) [reply polling_stations/apps/data_importers/management/commands/import_melton.py]: So the PR is not showing that I force-pushed anything, but the branch has updated and the print(record) is gone... I guess it's related to whatever synchronization issues are happening at the minute.

**Pull request merged**